templates/home/api.html

```diff
@@ -21,121 +21,147 @@
 	<dd>Template <!-- Common design patter --></dd>
 
 	<dt>Last review</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 
 	<dt>Conforming to</dt>
-	<dd>Going to be: Content and IA spec 2.0</dd>
+	<dd>Going to be: Content and IA spec 3.0</dd>
 
 	<dt>Specification</dt>
-	<dd><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/templates-detailed-specifications/home-page.html">Home page for Canada.ca</a>
+	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
 </dl>
 
 <h2>Technical specification</h2>
```

Comment:
Add a new section that list all the component on the home page, like:

```suggestion
<h2>Technical specification</h2>

<h3>Component list and latest version</h3>
<ul>
  <li><a href="#home-page-banner">Home page banner</a> - Version 2.0</li>
  <li>Most requested - Version 1.0</li>
  <li>Service and information - Version 1.0</li>
  <li>Focus on - Version 1.0</li>
  <li><a href="#your-government">Your government</a> - Version 2.0</li>
  <li>Government initiatives - Version 1.0</li>
</ul>

<h3 id="home-page-banner">Home page banner</h3>
...
<h3 id="your-government">Your government</h3>
...

```

Reply:
New section added as suggested. Thank you!